src/components/ProgressArray.js

```diff
@@ -23,20 +23,30 @@ export default class ProgressArray extends React.Component {
   }
 }
 
+const defaultProgressStyle = {
+  display: 'flex',
+  justifyContent: 'center',
+  maxWidth: '100%',
+  flexWrap: 'row',
+  position: 'absolute',
```

Comment:
I think using position: absolute and display flex together doesn't really work.. For the rest of the parameters I'm pretty sure we could just leave display: flex.

Reply:
It works! It can work simultaneously.

- display flex  + flexWrap row = it get all progress bar side by side
- position absolute = makes the progress bar in the top of the parent (story container)